### DISCUSSION TITLE
How to speed up the initialization of wavesurfer?

### DISCUSSION THREAD

**Discussion by baranowskyi** [katspaugh/wavesurfer.js#3795] [Q&A]
Hello. I am using pre-decoded peaks for wavesurfer. I only use wavesurfer for visual wave drawing, but it takes about 7 seconds for the first drawing. The examples on your site draw fast with my peak data. How to speed up the drawing of the wave? 

P.S. peak data has 8000 elements. I have used and very little, but the situation is the same.

**Comment by katspaugh** [comment]
If the same data is rendered fast in the examples, it must be something with your code.

**Reply by baranowskyi** [reply]
Yes. It's my fault. I'm using Nuxt and when I'm in development mode it runs slow, but when I'm in production mode it runs fast.
